v3/src/components/graph/models/graph-content-model.ts

```diff
@@ -161,6 +161,12 @@ export const GraphContentModel = DataDisplayContentModel
       return (self.plotType === 'scatterPlot' && ['left', 'bottom'].includes(place)) ||
              (self.pointDisplayType === "histogram")
     },
+    axisShouldShowZeroLine(place: AxisPlace) {
+      const otherPlace = place === "left" ? "bottom" : "left"
+      const otherPlaceRole = axisPlaceToAttrRole[otherPlace]
+      const attributeID = self.dataConfiguration.attributeID(otherPlaceRole)
+      return (self.plotType === 'dotPlot' && ['left', 'bottom'].includes(place)) && !!attributeID
```

Comment:
I think you want:
return (self.plotType === 'dotPlot' && ['left', 'bottom'].includes(place)) && self.pointDisplayType === "bars"